examples/platform/silabs/OTAConfig.cpp

```diff
@@ -18,6 +18,8 @@
 
 #include "OTAConfig.h"
 
+#ifndef SIWX_917
+
 #include "application_properties.h"
 #include <app/server/Server.h>
```

Comment:
I would expect that you still need `app/server/Server.h`with  917 too. The rest you are right, it currently only applies to EFR32 platform

Reply:
ok will add for 917 and verify once.

Reply:
I have added `app/server/Server.h `for 917.